Documentation Standard Updates (#15)
fivetran/dbt_google_play: README.md

</p>

# Google Play Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_google_play/))
## What does this dbt package do?
- Produces modeled tables that leverage Google Play data from [Fivetran's connector](https://fivetran.com/docs/applications/google-play) in the format described by [this ERD](https://fivetran.com/docs/applications/google-play#schemainformation) and build off the output of our [Google Play source package](https://github.com/fivetran/dbt_google_play_source).
- Enables you to better understand your Google Play app performance metrics at different granularities. It achieves this by:

> - Providing intuitive reporting at the App Version, OS Version, Device Type, Country, Overview, and Product (Subscription + In-App Purchase) levels
> - Aggregates all relevant application metrics into each of the reporting levels above

Author: ```suggestion
  - Providing intuitive reporting at the App Version, OS Version, Device Type, Country, Overview, and Product (Subscription + In-App Purchase) levels
  - Aggregates all relevant application metrics into each of the reporting levels above
```

- Generates a comprehensive data dictionary of your source and modeled Google Play data through the [dbt docs site](https://fivetran.github.io/dbt_google_play/).

> The following table provides a detailed list of all models materialized within this package by default.
> > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_google_play/#!/overview?g_v=1).
> 
> | **model**                  | **description**                                                                                                                                               |

Author: ```suggestion
The following table provides a detailed list of all tables materialized within this package by default.
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_google_play/#!/overview?g_v=1).

| **Table**                  | **Description**                                                                                                                                               |
```

</p>

# Google Play Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_google_play/))
## What does this dbt package do?
- Produces modeled tables that leverage Google Play data from [Fivetran's connector](https://fivetran.com/docs/applications/google-play) in the format described by [this ERD](https://fivetran.com/docs/applications/google-play#schemainformation) and build off the output of our [Google Play source package](https://github.com/fivetran/dbt_google_play_source).
- Enables you to better understand your Google Play app performance metrics at different granularities. It achieves this by:
  - Providing intuitive reporting at the App Version, OS Version, Device Type, Country, Overview, and Product (Subscription + In-App Purchase) levels
  - Aggregates all relevant application metrics into each of the reporting levels above
- Generates a comprehensive data dictionary of your source and modeled Google Play data through the [dbt docs site](https://fivetran.github.io/dbt_google_play/).

> 

Author: ```suggestion

<!--section=“google_play_transformation_model"-->

```

| [google_play__overview_report](https://fivetran.github.io/dbt_google_play/#!/model/model.google_play.google_play__overview_report)   | Each record represents daily installs, crashes and ANRs, store performance metrics, and ratings by app.                            |
| [google_play__finance_report](https://fivetran.github.io/dbt_google_play/#!/model/model.google_play.google_play__finance_report) | Each record represents daily subscriptions, purchases, and different kinds of revenue by product and country.                          |

> 

Author: ```suggestion

<!--section-end-->
```
